src/commands/match.ts

```diff
@@ -872,7 +872,7 @@ async function matchLines(outFile, params, lines, flags) {
       var gisRef:SharedStreetsReference = forwardReference(line);
 			
       matchForward = await matcher.matchGeom(line);
-      if(matchForward && matchForward.score < matcher.searchRadius * 2) {
```

Comment:
This check of the `score` is somewhat arbitrary - it is calculated with a combination of the difference between the start and end points with the matched traces and the difference between the original length and the matched length.

From our point of view, I reckon if we successfully got a match that was outputted by OSRM, then we are happy. Dealing with failed matches is a pain, so we don't want to have these arbitrary constraints on the output